Samples/BackgroundBlur/BackgroundBlur/TransformAsync_IMFAsyncCallback.cpp

```diff
@@ -0,0 +1,119 @@
+#include "TransformAsync.h"
+#include <format>
+
+int g_now;
```

Comment:
>g_now [](http://example.com/codeflow?start=4&length=5)

 I would avoid globals variables if possible since there could be multiple instances of the class

Reply:
This global is separate from the TransformAsync, it's used only by FrameThreadProc to measure the time between calls to the allocator's NotifyRelease.